cmd/ccx-notification-service/main.go

```diff
@@ -84,7 +84,7 @@ func main() {
 		log.Err(err).Msg(loadConfigurationMessage)
 		os.Exit(ExitStatusConfiguration)
 	}
-
+	log.Error().Msg("TEST ERROR FOR NOTIFICATION SERVICE")
```

Comment:
remove, there will be errors already, so it's not needed

Reply:
Ok removed